spec/models/chargeback_container_project_spec.rb

```diff
@@ -206,7 +208,7 @@
 
   context "gets rate from enterprise" do
     let(:options) { base_options.merge(:interval => 'monthly', :entity_id => @project.id, :tag => nil) }
-    let(:miq_enterprise) { FactoryGirl.create(:miq_enterprise) }
+    let(:miq_enterprise) { MiqEnterprise.first || FactoryGirl.create(:miq_enterprise) }
```

Comment:
If you seed it in the before, it can't not exist, so you can remove the factory create.